# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1398880727 -7200
#      Wed Apr 30 19:58:47 2014 +0200
# Branch gui-release
# Node ID d6b962aa2e7611def16b5e273fe1d6a379931192
# Parent  f31fde98c8728ed3d4ff2b72bff0bfa6251a378c
fix compiling when scintilla is not available

* main-window.cc (connect_visibility_changed): add #ifdef around editor_window

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -1001,18 +1001,20 @@ main_window::write_settings (void)
 // Connecting the signals emitted when the visibility of a widget changes.
 // This has to be done after the window is shown (see octave-gui.cc)
 void
 main_window::connect_visibility_changed (void)
 {
   foreach (octave_dock_widget *widget, dock_widget_list ())
     widget->connect_visibility_changed ();
 
+#ifdef HAVE_QSCINTILLA
   // Main window completely shown, determine whether to create an empty script
   editor_window->empty_script (true, false);
+#endif
 }
 
 void
 main_window::copyClipboard (void)
 {
   if (_current_directory_combo_box->hasFocus ())
     {
       QLineEdit * edit = _current_directory_combo_box->lineEdit ();
